File: examples/notes/create/go/createANote.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/notes"
)

// Create a new note
func CreateAnote() {
	notesService := notes.NewNotesService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := notesService.Create(&notes.CreateRequest{
		Text:  "This is my note",
		Title: "New Note",
	})
	fmt.Println(rsp, err)
}

File: examples/notes/create/node/createANote.js
const { NotesService } = require("m3o/notes");

// Create a new note
async function createAnote() {
  let notesService = new NotesService(process.env.MICRO_API_TOKEN);
  let rsp = await notesService.create({
    text: "This is my note",
    title: "New Note",
  });
  console.log(rsp);
}

createAnote();
